Automattic/wp-calypso
# Fix: Remove marketing tool card for Bill.com as the link 404s

Closes Automattic/martech#2253


## Proposed Changes

* When clicking the "Manage bills and invoices" link on the "Bill.com" card under marketing > tools > business tools, the user is taken to a page with a 404 status.
* I'm removing this card entirely for now, as I can't find much context on this. We can always re-add it in the future.

## Testing Instructions



* Go to `http://calypso.localhost:3000/marketing/business-tools/<testsite.wordpress.com>`. Verify that you don't see the Bill.com card anymore, and that there are no 404ing resources or console errors related to this change.

## Pre-merge Checklist
